tests/commands/install_uninstall_test.py

```diff
@@ -532,7 +532,7 @@ def test_pre_push_integration_failing(tempdir_factory, store):
         assert retc == 1
         assert 'Failing hook' in output
         assert 'Failed' in output
-        assert 'foo zzz' in output  # both filenames should be printed
+        assert 'zzz foo' in output  # both filenames should be printed
```

Comment:
lol i messed with the fixed random seed until only one test failed rather than a bunch

Reply:
lol nice

Reply:
derp, apparently this is actually passing as-is (i needed to `pip install` into my venv again)